parser/src/layout.rs

```diff
@@ -318,10 +313,12 @@ impl<'input, Tokens> Layout<'input, Tokens>
                     // ```
                     // `let x = 1 in {{ a; b }}` and not `{{ (let x = 1 in a) ; b }}`
                     let offside = Offside::new(location, Context::Block { emit_semi: false });
-                    self.indent_levels.push(offside)?;
+                    if let Err(err) = self.indent_levels.push(offside) {
```

Comment:
Yeah, will try that - kind of a pain not to have `?` for `Option`... @nikomatsakis - where are our monads! :D